<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModeForm</class>
 <widget class="QWidget" name="ModeForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>45</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>45</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>45</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#ModeForm
{
background: rgba(54, 57, 63, 200);
border-radius: 22px;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>8</number>
   </property>
   <property name="leftMargin">
    <number>12</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>12</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QPushButton" name="chatBtn">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>32</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	background: #4a4a4a;
	color: white;
	border: none;
	border-radius: 16px;
	font-size: 13px;
}

QPushButton:hover
{
	background: #5a5a5a;
}

QPushButton:checked
{
	background: #0078d4;
}</string>
     </property>
     <property name="text">
      <string>闲聊</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="superviseBtn">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>32</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	background: #4a4a4a;
	color: white;
	border: none;
	border-radius: 16px;
	font-size: 13px;
}

QPushButton:hover
{
	background: #5a5a5a;
}

QPushButton:checked
{
	background: #0078d4;
}</string>
     </property>
     <property name="text">
      <string>监督</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="teachBtn">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>32</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	background: #4a4a4a;
	color: white;
	border: none;
	border-radius: 16px;
	font-size: 13px;
}

QPushButton:hover
{
	background: #5a5a5a;
}

QPushButton:checked
{
	background: #0078d4;
}</string>
     </property>
     <property name="text">
      <string>教学</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="standbyBtn">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>32</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	background: #4a4a4a;
	color: white;
	border: none;
	border-radius: 16px;
	font-size: 13px;
}

QPushButton:hover
{
	background: #5a5a5a;
}

QPushButton:checked
{
	background: #0078d4;
}</string>
     </property>
     <property name="text">
      <string>待机</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <connections/>
</ui>
